RELEASENOTES-1.4.docu

```diff
@@ -521,4 +521,12 @@
       This fix is only enabled by default with Simics API version 7 or above.
       With version 6 or below it must be explicitly enabled by passing
       <tt>--no-compat=shared_logs_on_device</tt> to DMLC.</add-note></build-id>
+  <build-id value="next"><add-note> Bank instrumentation callbacks are now
+      issued from the <tt>read</tt> and <tt>write</tt> methods instead of the
+      <tt>transaction_access</tt>/<tt>io_memory_access</tt> method of
+      banks. This means that breakpoints on banks will trigger also on
+      reads or writes initiated within the device. This also means
+      that if <tt>read</tt> or <tt>write</tt> is overridden in a bank,
+      then instrumentation callbacks will only be called if they
+      call <tt>default()</tt>.</add-note></build-id>
```

Comment:
This perhaps deserves a discussion, there are probably valid use cases for read/write overrides where this change will be a regression. I don't know what to do about that.